README.md

```diff
@@ -35,6 +35,19 @@ services:
       - DISCORD_USER_ID= # Discord user ID of the user you want Aoede to follow
 ```
 
+**Docker**:
+```env
+# .env
+ENV DISCORD_TOKEN=
+ENV SPOTIFY_USERNAME=
+ENV SPOTIFY_PASSWORD=
+ENV DISCORD_USER_ID=
+```
+
+```bash
+docker run -d --env-file .env codetheweb/aoede
```

Comment:
Maybe add `--rm` to automatically remove the container? Don't know

Reply:
Sure, will do :)

Reply:
Fixed in 929c989798a09cd47f5a690944d22443ecf92f44